Core.ApplicationServices/GDPR/IDataProcessingRegistrationApplicationService.cs

```diff
@@ -25,6 +26,8 @@ public interface IDataProcessingRegistrationApplicationService
         Result<IEnumerable<ItSystem>, OperationError> GetSystemsWhichCanBeAssigned(int id, string nameQuery, int pageSize);
         Result<ItSystem, OperationError> AssignSystem(int id, int systemId);
         Result<ItSystem, OperationError> RemoveSystem(int id, int systemId);
+        Result<DataProcessingRegistration, OperationError> UpdateOversightInterval(int id, YearMonthIntervalOption? yesNoIrrelevantOption);
```

Comment:
Den værdi skal ikke være nullable og navngivningen skal være i orden f.eks. oversightInterval... i stedet for det andet rod